action.yml

```
@@ -1,5 +1,5 @@
 name: C/C++ Linter
-description: Lint C/C++ code with clang-format and clang-tidy then post annotations and comments with faulty results.
+description: Lint C/C++ code with clang-format and clang-tidy then post annotations, comments and step summary with faulty results.
```

1. missing a comma
2. results no longer focus only on "faulty results"
```suggestion
description: Lint C/C++ code with clang-format and clang-tidy then post annotations, comments, and step summary with results.
```